dali/pipeline/operators/decoder/nvjpeg_helper.h

```diff
@@ -113,6 +126,91 @@ void HostFallback(const uint8_t *data, int size, DALIImageType image_type, uint8
   kernels::copy<StorageType, kernels::StorageCPU>(output_buffer, decoded.get(), h * w * c, stream);
 }
 
+
+inline void WarmUpNvJPEG(nvjpegHandle_t handle, StateNvJPEG& state, nvjpegJpegDecoder_t decoder,
+                  nvjpegOutputFormat_t output_format) {
```

Comment:
This function is quite costly and there doesn't seem to be any advantage of keeping it inlined (or in h file). Please move implementation to cc file.

Additionally, is there any way to say, that the nvjpeg has beed warmed up?